components/input.css

```diff
@@ -109,15 +113,15 @@
     padding: 0 4px;
     white-space: nowrap;
     border: none;
+    color: rgba(var(--center-channel-color-rgb), 0.64);
```

Comment:
Here could we just use `var(--center-channel-color-64)`?

Reply:
We are moving away from using the static variables provided by `applyTheme`, this way we don't need to have a different CSS variable every time we want a slightly lighter/darker version of one of the theme colours.

Reply:
Good to know. Thanks!